x-pack/packages/kbn-elastic-assistant/impl/assistant/prompt_editor/system_prompt/index.tsx

```diff
@@ -31,9 +32,8 @@ const SystemPromptComponent: React.FC<Props> = ({
   onSystemPromptSelectionChange,
   setIsSettingsModalVisible,
   isFlyoutMode,
+  allSystemPrompts,
```

Comment:
```suggestion
  allPrompts,
```
lets rename it here too, i think we're passing `allPrompts` without filtering

Reply:
I added filtering.